lib/ruby_lsp/requests/workspace_symbol.rb

```diff
@@ -53,6 +43,23 @@ def perform
           )
         end
       end
+
+      private
+
+      def fuzzy_search
+        @index.fuzzy_search(@query) do |entry|
+          file_path = entry.uri.raw_path
```

Comment:
I am not sure if this is the right approach, we could rather cache this on entry level instead of skipping unscapping parser